guides/release/reference/accessibility.md

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
This file needs to have the same name as the deleted file in order for the redirect to work.

The goal of the redirect is that `https://guides.emberjs.com/release/reference/accessibility-guide/` does not become a dead link.

Reply:
oh that's a silly mistake. Thanks for catching it!